### DISCUSSION TITLE
NCMD Rebirth Stops Publishing

### DISCUSSION THREAD

**Discussion by m2b** [thomassorensen2650/node-red-contrib-mqtt-sparkplug-plus#44] [Q&A] [answered]
I have observed that when an NCMD rebirth is issued, the Sparkplug Out node stops sending messages for up to a minute.   Is this expected or can it be tweaked.  I am pretty convinced that it is even discarding any messages coming from other nodes from the flow during this period.

**Comment by thomassorensen2650** [comment]
This is the first I've heart of this issue. Are you getting any error messages in Node-Red? and are the messages buffered and send later?

A rebirth to a  Sparkplug Device should not affect the Sparkplug Out node in any way.

**Reply by m2b** [reply, marked as answer]
My bad, I looked at logging and the host was rebooting coincidentally when the NCMD was sent.
